src/modules/hip/hip_color_model_conversions.cpp

```diff
@@ -86,47 +86,47 @@ color_temperature_hip_batch (   Rpp8u* srcPtr, Rpp8u* dstPtr, rpp::Handle& handl
 RppStatus
 vignette_hip(Rpp8u* srcPtr, RppiSize srcSize, Rpp8u* dstPtr, float stdDev, RppiChnFormat chnFormat, unsigned int channel, rpp::Handle& handle)
 {
-    int ctr=0;
-    cl_kernel theKernel;
-    cl_program theProgram;
-    if(chnFormat == RPPI_CHN_PLANAR)
-    {    
-        std::vector<size_t> vld{32, 32, 1};
-        std::vector<size_t> vgd{(srcSize.width + 31) & ~31, (srcSize.height + 31) & ~31, channel};
-        handle.AddKernel("", "", "vignette.cpp", "vignette_pln", vld, vgd, "")(srcPtr,
-                                                                            dstPtr,
-                                                                            srcSize.height,
-                                                                            srcSize.width,
-                                                                            channel,
-                                                                            stdDev
-                                                                            );
-    } 
-    else
-    {
-        std::vector<size_t> vld{32, 32, 1};
-        std::vector<size_t> vgd{(srcSize.width + 31) & ~31, (srcSize.height + 31) & ~31,channel};
-        handle.AddKernel("", "", "vignette.cpp", "vignette_pkd", vld, vgd, "")(srcPtr,
-                                                                            dstPtr,
-                                                                            srcSize.height,
-                                                                            srcSize.width,
-                                                                            channel,
-                                                                            stdDev
-                                                                            );
-    }
+    // int ctr=0;
```

Comment:
if we are not using these implementations anymore, it is better to remove the entire function

Reply:
For all the commented functions in this file